app/controllers/returns/credits/ConfirmPackagingCreditController.scala

```diff
@@ -16,37 +16,52 @@
 
 package controllers.returns.credits
 
-import connectors.ExportCreditsConnector
+import connectors.CalculateCreditsConnector
 import controllers.actions._
-import models.Mode
-import models.Mode.NormalMode
-import navigation.ReturnsJourneyNavigator
+import models.CreditBalance
+import models.requests.DataRequest
 import play.api.i18n.{I18nSupport, MessagesApi}
-import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
-import views.html.returns.credits.ConfirmPackagingCreditView
+import viewmodels.{PrintBigDecimal, PrintLong}
+import views.html.returns.credits.{ConfirmPackagingCreditView, TooMuchCreditClaimedView}
 
 import javax.inject.Inject
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 class ConfirmPackagingCreditController @Inject()(
-                                                  override val messagesApi: MessagesApi,
-                                                  exportCreditsConnector: ExportCreditsConnector,
-                                                  identify: IdentifierAction,
-                                                  getData: DataRetrievalAction,
-                                                  requireData: DataRequiredAction,
-                                                  val controllerComponents: MessagesControllerComponents,
-                                                  view: ConfirmPackagingCreditView,
-  returnsNavigator: ReturnsJourneyNavigator
+  override val messagesApi: MessagesApi,
+  creditConnector: CalculateCreditsConnector,
+  identify: IdentifierAction,
+  getData: DataRetrievalAction,
+  requireData: DataRequiredAction,
+  val controllerComponents: MessagesControllerComponents,
+  view: ConfirmPackagingCreditView,
```

Comment:
can we rename this view, as there is 2

Reply:
cannot see another ConfirmPackagingCreditView.scala.html. do you know where it is?

Reply:
ok, done